docs/source/backup/specification.rst

```diff
@@ -155,6 +164,15 @@ Under the node directory each table version has it's directory, the path is stru
 The directory contains all the table files.
 Some files may be used in more than one backup.
 
+You may also find that some of those files have the snapshot tag suffix (e.g. ``.sm_20210809095541UTC``).
+Even though sstables (data files) are immutable by nature, it's sometimes possible to have two different sstables originating
+from the same table and with the same node ID. In this case, older versions of sstable are appended with the snapshot tag
```

Comment:
Worthwhile mentioning this can only happen if you replace a node. 
It will give the issue context.

Also, once https://github.com/scylladb/scylladb/issues/6403 is fixed, two nodes won't have the same ID (just FYI, there is no need to include it here)